lib/Doctrine/Common/Annotations/AnnotationReader.php

```diff
@@ -292,6 +292,38 @@ public function getMethodAnnotation(ReflectionMethod $method, $annotationName)
         return null;
     }
 
+    /**
+     * {@inheritDoc}
+     */
+    public function getConstantAnnotations(\ReflectionClassConstant $constant)
```

Comment:
Please add php-7.1-compatible type hints everywhere you can.

Reply:
I'm keeping exactly the same API as for properties methods. I believe the consistency is more useful here than perfection. However, this can be improved when portin these changes to the master branch.

Reply:
I don't know the option of other maintainers, but IMO, since we are moving towards type hints everywhere, there is no point in making the BC-break that will occur when we finally add them bigger. I think consistency is good, except when it's consistency with a deprecated thing we are trying to move away from.

Reply:
I would break them all at once in 2.0, but as you wish.